### PR TITLE
[stable/cluster-autoscaler] support fullnameOverride

### PR DESCRIPTION
**Pull request by erks**
#### What this PR does / why we need it:

Support `fullnameOverride` and be consistent with other charts.

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.8.0
+version: 0.8.1
```
Comment (davidkarlsen) [inline stable/cluster-autoscaler/Chart.yaml]:
```suggestion
version: 0.9.0
```
due new feature

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/cluster-autoscaler/Chart.yaml (hunk 1 above)
Comment (erks) [reply stable/cluster-autoscaler/Chart.yaml]: done

**Review by davidkarlsen — APPROVED** [review]

**Comment by erks** [comment]
@davidkarlsen looks like there was a version conflict on master. I bumped this to `0.10.0` (from `0.9.0` on master). Please `lgtm` again.

**Comment by erks** [comment]
@davidkarlsen another bump.

**Pull request closed without merging**